packages/storybook/package.json

```diff
@@ -48,7 +48,7 @@
     "remark-gfm": "^4.0.0",
```

Comment:
Turbosnap for the chromatic tests is showing zero snaps taken:
![image](https://github.com/user-attachments/assets/206a3370-cc55-43df-82cf-6c13a7daef9a)

We want to make sure there are not regressions; bump the guid in preview.js to make sure new snapshots are taken